api/app/db/models.py

```diff
@@ -187,3 +187,11 @@ class NoonForecasts(Base):
     danger_rating = Column(Integer, nullable=False)
     created_at = Column(TIMESTAMP(timezone=True), nullable=False,
                         default=datetime.datetime.now(tz=timezone.utc))
+
+    def __str__(self):
+        return (
+            'weather_date:{self.weather_date}, '
+            'created_at:{self.created_at}, '
+            'temp={self.temperature}, '
+            'ffmc={self.ffmc}'
```

Comment:
did you mean to print rh instead of ffmc?

Reply:
No, I wanted to check what one of the fire indexes would look like